feed.xml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I think we might not neet this page too.
You can remove all blog entries also :)